modules/nw-sriov-networknodepolicy-object.adoc

```diff
@@ -76,11 +76,13 @@ If the `isRdma` parameter is set to `true`, you can continue to use the RDMA-ena
 +
 Set `isRdma` to `true` and additionally set `needVhostNet` to `true` to configure a Mellanox NIC for use with Fast Datapath DPDK applications.
 
-<17> Optional: The link type for the VFs. You can specify one of the following values: `eth` or `ib`. Specify `eth` for Ethernet or `ib` for InfiniBand. The default value is `eth`.
-
-<18> Optional: To enable hardware offloading, the 'eSwitchMode' field must be set to `"switchdev"`.
+<17> Optional: The link type for the VFs. The default value is `eth` for Ethernet. Change this value to 'ib' for InfiniBand.
 +
 When `linkType` is set to `ib`, `isRdma` is automatically set to `true` by the SR-IOV Network Operator webhook. When `linkType` is set to `ib`, `deviceType` should not be set to `vfio-pci`.
++
+Do not set linkType to 'eth' for SriovNetworkNodePolicy, because this can lead to an incorrect number of available devices reported by the device plugin.
```

Comment:
`linkType`
    `SriovNetworkNodePolicy`
(both should be in backticks)

watch the markdown of 'ib' , 'eth' , `switchdev`, and others. They should also be in backticks and not other formatting, so that the formatting renders per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

We use plug-in instead of plugin